socorro/cron/jobs/update_signatures.py

```diff
@@ -98,8 +99,8 @@ def run(self, end_datetime):
 
         # Do a super search and get the signature, buildid, and date processed for
         # every crash in the range
-        all_fields = SuperSearchFields(config=self.config).get()
-        api = SuperSearch(config=self.config)
+        all_fields = SuperSearchFieldsData().get()
```

Comment:
`SuperSearchFieldsData` is **just** the fields now and doesn't need configuration to instantiate.